webots_ros2_tutorials/protos/RobotSense.proto

```diff
@@ -0,0 +1,210 @@
+#VRML_SIM R2020b utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0
+# tags: static
+# More info here: https://www.youtube.com/c/soft-illusion
+# Robotsense
+
+PROTO RobotSense [
+    field SFVec3f    translation  0 0.0198 0
+    field SFRotation rotation     0 1 0 2.3495
+    field SFString   controller   "<extern>"
+  ]
+  {
+    Robot {
+    translation IS translation
```

Comment:
Sorry, but we are still missing two spaces in the indentation of this block here.

Reply:
See comment below.

Reply:
Here above I see `translation` aligned with `Robot` on the upper line, which is not the case for the other protos, E-puck and Khepera4 (see your screenshots) where `translation` is indented by two spaces from `Robot`.

Reply:
Oh I see ...
Now it is fine ...

My browser did not show the above message as new one